### PR TITLE
Spec updates

### PR DESCRIPTION
**Pull request by rluzynski**
Having received the "go ahead" answer from Igor I create this pull request. Also please verify if the patches are correct.

### CODE REVIEW

File: gnome-abrt.spec.in [hunk 1]
```diff
@@ -77,36 +76,27 @@ desktop-file-install \
...
-%postun
-if [ $1 -eq 0 ] ; then
-    touch --no-create %{_datadir}/icons/hicolor &>/dev/null
-    gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
-fi
-
-
-%posttrans
-gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
-
```
Comment (mhabrnal) [inline gnome-abrt.spec.in]: This change of post scriplets are relevant for fedoras > 27. You should add conditional

%if 0%{?fedora} > 27
# post scriplets are not needed
%else
...
%endif

**Comment by mhabrnal** [comment]
Otherwise looks good

File: gnome-abrt.spec.in (hunk 1 above)
Comment (ignatenkobrain) [reply gnome-abrt.spec.in]: @mhabrnal no, this is wrong. You should do `%if 0%{?rhel} && 0%{?rhel} <= 7`..

But I would remove it entirely.

Comment (mhabrnal) [reply gnome-abrt.spec.in]: @ignatenkobrain  so, these post scriplets are no longer needed in Fedora 26 and Fedora 27?

Comment (ignatenkobrain) [reply gnome-abrt.spec.in]: just FYI, F26 is dead.. Really.

But yes, they are not needed since F26/

**Pull request merged**